### PR TITLE
coded file

### PR DESCRIPTION
**Pull request by Szpytowski**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+*/
+// now with for each
+function getAverage(marks) {
+  let totalScore = 0;
+  marks.forEach(function (mark) {
+    totalScore += mark;
+  });
+  let averageScore = totalScore / marks.length;
+
+  return Math.floor(averageScore);
```
Comment (Solnick) [inline js/app.js]: Why do you want to use the number rounding here?

**Review by Solnick — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 1 above)
Comment (Szpytowski) [reply js/app.js]: I was thinking in terms of the polish points system, "trojka, czworka ..." and wanted to round it up. I changed it

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+*/
+// now with for each
+function getAverage(marks) {
+  let totalScore = 0;
+  marks.forEach(function (mark) {
+    totalScore += mark;
+  });
+  let averageScore = totalScore / marks.length;
+
+  return (averageScore);
```
Comment (Solnick) [inline js/app.js]:
```suggestion
  return totalScore / marks.length;
```

File: js/app.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+
+// we have to filter out the positive numbers
+// we have to get the postive numbers and sumthem
+*/
+// now turn it into forEach
+
+function positiveNumFilter(numbers) {
+  let positiveNumbers = [];
```
Comment (Solnick) [inline js/app.js]: can be const

File: js/app.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  let totalScore = 0;
+  marks.forEach(function (mark) {
+    totalScore += mark;
+  });
+  let averageScore = totalScore / marks.length;
+
+  return (averageScore);
+}
+
+let marksArray = [4, 5, 3, 4, 5];
```
Comment (Solnick) [inline js/app.js]: can be const

**Review by Solnick — CHANGES_REQUESTED** [review]

**Review by Solnick — APPROVED** [review]

**Pull request merged**